src/ray/common/task/scheduling_resources.cc

```diff
@@ -231,6 +231,14 @@ void ResourceSet::AddResources(const ResourceSet &other) {
 void ResourceSet::CommitBundleResources(const PlacementGroupID &group_id,
                                         const int bundle_index,
                                         const ResourceSet &other) {
+  // Add a implicity label to avoid the default schedule issue.
+  // https://github.com/ray-project/ray/issues/11692.
+  const std::string &implicity_label =
+      FormatPlacementGroupImplicityResource(group_id, bundle_index);
+  const FractionalResourceQuantity &resource_capacity =
```

Comment:
It is currently not removed when the bundle is returned right?

Reply:
yes, I have implements this in the new commit, maybe you can review it, thank you~